### ISSUE TITLE
Investigate whether Integration tests are flaky because of running two tests in parallel

### ISSUE THREAD

**Issue by balajisankar15** [aws/sagemaker-distribution#36]
While doing a minor version release(0.4.0), some integration tests(Pytorch and tensorflow) were failing. Current test set up is running two tests in parallel. We tried changing that to 1 and the tests succeeded afterwards.

Determine if this is a symptom of bad test code (or) a problem with the environment in which the tests were executed.

**Comment by balajisankar15** [comment]
Confirmed that this is an environment problem (Available space_
